### PR TITLE
fix(3151428244): rename colors

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

* Refactor color names to more meaningful names created by CSS colors name generator.
* Remove unused colors

Solves #[3151428244](https://starkware.monday.com/boards/2847524005/pulses/3151428244)

---

### Checklist

- [x] Changes have been done against `dev` branch, and PR does not conflict.
- [x] Manually tests of the main Application flows are done and passed.
- [ ] New unit / functional tests have been added (whenever applicable).
- [x] Automation tests are passing in local environment.
- [ ] Docs have been updated (whenever relevant).
- [x] PR title is follow the [Conventional Commits](https://www.conventionalcommits.org/) convention: `<type>[optional scope]: <description>`, e.g: `fix: prevent racing of requests`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starkgate-frontend/292)

### CODE REVIEW

**Review by galuzan-starkware — APPROVED** [review]

File: src/styles/colors.module.scss [hunk 1]
```diff
@@ -1,45 +1,38 @@
...
+$--color-american-blue: #3b3e76;
+$--color-purple-navy: #494a81;
+$--color-ube: #8b8dc2;
+$--color-ceil: #999bc5;
+$--color-orange-soda: #f6643c;
+$--color-very-light-azure: #69b7f9;
+$--color-dodger-blue: #1595ff;
+$--color-granite-gray: #5f666c;
+$--color-auro-metal-saurus: #667085;
+$--color-spanish-green: #039855;
```
Comment (yairans-starkware) [inline src/styles/colors.module.scss:18]: unused color

**Pull request merged**